release-notes/2022-11/README.md

```diff
@@ -0,0 +1,183 @@
+# SystemLink Enterprise 2022-11 Release Notes
+
+The 2022-11 release bundle for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release please review past refer to past release notes to ensure all required configuration changes are addressed.
+
+## Upgrading from the 2022-09 Release to the 2022-11 Release
+
+Due to breaking changes in the top level SystemLink and SystemLink-Admin Helm chart the following procedure is required to upgrade the application. Note, this upgrade will cause any currently executing notebooks to fail.
+
+1. Stage your workspace to upgrade the `systemlink-admin` and `systemlink` helm charts to the 2022-11 release.
+
+1. Run the Helm upgrade command on the `systemlink` chart to remove existing Argo CRDs. This must be done before upgrading the `systemlink-admin` chart. This command should include all flags and value file references you would expect to use for a typical upgrade of the application.
+
+    **Note:** This wil result in an expected failure: `UPGRADE FAILED: unable to recognize "": no matches for kind "WorkflowTemplate" in version "argoproj.io/v1alpha1"`. This error can be ignored.
```

Comment:
```suggestion
    **Note:** This will result in an expected failure: `UPGRADE FAILED: unable to recognize "": no matches for kind "WorkflowTemplate" in version "argoproj.io/v1alpha1"`. Ignore this error.
```